.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -40,45 +40,46 @@ body:
       description:  What JASP analysis are you seeing the problem on?
       multiple: true
       options:
-        - jaspDescriptives
-        - jaspAnova
-        - jaspFactor
-        - jaspFrequencies
-        - jaspRegression
-        - jaspTTests
-        - jaspMixedModels
-        - jaspAudit
-        - jaspBain
-        - jaspNetwork
-        - jaspSem
-        - jaspMachineLearning
-        - jaspSummaryStatistics
-        - jaspMetaAnalysis
-        - jaspDistributions
-        - jaspEquivalenceTTests
-        - jaspJags
-        - jaspReliability
-        - jaspVisualModeling
-        - jaspLearnBayes
-        - jaspProphet
-        - jaspProcessControl
-        - jaspCircular
+        - Other
```

Comment:
I would set as first option 'Any' for bugs which are not concerning.a particular analysis. The 'Other' option can be added as last item, just in case a module is missing in the list.